src/siteconfig/tests/test_views.py

```diff
@@ -36,14 +36,14 @@ def test_SiteConfigUpdateOwn(self):
         """
 
         # requires staff member
-        self.assertRedirectsAdmin('config:site_config_update_own')
+        self.assertRedirectsLogin('config:site_config_update_own')
```

Comment:
I don't think there's a test for students? They should get 403 for this view.

Reply:
ill add some status code tests for ``SiteConfigViewTest``

Reply:
replaced this:
![image](https://user-images.githubusercontent.com/39788517/183224206-29273fbf-94ed-4fd4-a928-79d245eb8808.png)
with this (since this seems to be our standard way of testing status codes):
![image](https://user-images.githubusercontent.com/39788517/183224268-9712af20-eae6-434d-80e3-5726fe85e7bd.png)